src/client/social.radiatus.js

```diff
@@ -16,25 +16,21 @@
  * @param {WebSocket} webSocket Alternative webSocket implementation for tests
  **/
 var DEBUGLOGGING = false;
+var WS_URL = 'ws://localhost:8082/route/';
+//var WS_URL = 'wss://data.radiatus.io/route/';
```

Comment:
Good point. The code we had before was defacto hardcoded, because the DEBUG variable is never set or passed in anywhere. So this was just to clear up the code 

We do want some way to configure the endpoint (e.g.dist vs testing) and my issue here will track that
https://github.com/freedomjs/radiatus-providers/issues/16